src/routes/popup/agents/components/ao-yield/DateSelectorModal.tsx

```diff
@@ -30,9 +61,16 @@ interface DateSelectorScreenProps {
   onSelect: (startDate: Date, endDate: Date) => void;
   currentStartDate: Date | null;
   currentEndDate: Date | null;
+  runIndefinitely: boolean;
 }
 
-const DateSelectorScreen = ({ onClose, onSelect, currentStartDate, currentEndDate }: DateSelectorScreenProps) => {
+const DateSelectorScreen = ({
```

Comment:
Not sure implementing a custom date selector is a good idea. Probably easier and safer to find a library we can customize to fit our needs.